web/public/js/directives/cd-badge-cert/template.dust

```diff
@@ -0,0 +1,11 @@
+<div class="cd-badge-cert">
+  <h2>{{ $ctrl.badge.name }}</h2>
+  <div class="cd-badge-cert__info">
+    <div class="cd-badge-cert__image">
+      <img ng-src="{{ $ctrl.badge.imageUrl }}" />
+    </div>
+    <p class="cd-badge-cert__description" ng-bind-html="$ctrl.badge.consumerDescription | linky"></p>
```

Comment:
is consumerDescription manual user entry? If yes, ng-bind-html could be dangerous